### PR TITLE
[SA-Contrib-2018-077]: Apply recommended changes to password policy

### PR DESCRIPTION
**Pull request by fubarhouse**
This change is additional to #758 in that it introduces the recommended policy changes to the policies to prevent applicability to the vulnerability identified.

See #758 for a full breakdown and links to SA on drupal.org with this recommendation.

### CODE REVIEW

**Review by pandaskii — APPROVED** [review]
Happy to merge it after testing, thanks

**Comment by fubarhouse** [comment]
As discussed I will also submit the changes to Behat tests to reflect the above change.
This will later translate to the `7.x.2.x` branch.

**Review by pandaskii — APPROVED** [review]
Thanks for the updated testing feature

File: tests/behat/features/configuration/password_policy.feature [hunk 1]
```diff
@@ -21,6 +21,7 @@ Feature: Password Policy
     And the "warning" field should contain "7"
     And the "constraint_username" field should contain "1"
     And the "constraint_delay" field should contain "24"
+    And the "constraint_digit" field should contain "1"
```
Comment (pandaskii) [inline tests/behat/features/configuration/password_policy.feature]: Is the field name "constraint_digit_placement" instead of "constraint_digit"?

**Review by pandaskii — CHANGES_REQUESTED** [review]

File: tests/behat/features/configuration/password_policy.feature (hunk 1 above)
Comment (fubarhouse) [reply tests/behat/features/configuration/password_policy.feature]: image

**Review by pandaskii — APPROVED** [review]

File: tests/behat/features/configuration/password_policy.feature (hunk 1 above)
Comment (fubarhouse) [reply tests/behat/features/configuration/password_policy.feature]: My mistake - it's actually `constraint_digit_placement`.
I've just committed the changes.

**Comment by fubarhouse** [comment]
Behat tests will cover this one off completely - tests look like they will pass now.

**Comment by pandaskii** [comment]
Thanks, ready for release

**Pull request merged**